be/src/Project/Services/Catalog/Catalog.Api/v1/Categories/Controllers/CategoriesController.cs

```diff
@@ -51,7 +52,7 @@ public CategoriesController(ICategoriesService categoryService)
         [ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(PagedResults<IEnumerable<CategoryResponseModel>>))]
         [ProducesResponseType((int)HttpStatusCode.NoContent)]
         [ProducesResponseType((int)HttpStatusCode.UnprocessableEntity)]
-        public async Task<IActionResult> Get(string searchTerm, int? level, bool? leafOnly, int? pageIndex, int? itemsPerPage, string orderBy)
```

Comment:
**question:** Why we stop using async?

Reply:
Hi @SebaKas21, it makes no sense to use async if inside the method there is no await inside. It throws a warning in Visual Studio, makes building solution slower and execution of the code less performant.